engines/standard_tasks/app/models/standard_tasks/paper_reviewer_task.rb

```diff
@@ -6,6 +6,18 @@ def self.permitted_attributes
 
     register_task default_title: "Assign Reviewers", default_role: "editor"
 
+    include Invitable
+
+    def invitation_invited(invitation)
+      PaperReviewerMailer.delay.notify_invited({
+        invitation_id: invitation.id
+      })
+    end
+
+    def invitation_accepted(invitation)
+      TaskRoleUpdater.new(self, invitation.invitee_id, PaperRole::REVIEWER).update
+    end
+
     def array_attributes
       super + [:reviewer_ids]
```

Comment:
yep sounds like a good pattern! This was the first time I implemented an `Invitable` so haven't quite felt the various pains yet. What do you think is the best way to go forward?